nibabel/tests/test_scripts.py

```diff
@@ -67,6 +67,14 @@ def check_nib_ls_example4d(opts=[], hdrs_str="", other_str=""):
     assert_equal(fname, stdout[:len(fname)])
     assert_re_in(expected_re, stdout[len(fname):])
 
+
+def check_nib_diff_examples():
+    fnames2 = [pjoin(DATA_PATH, f)
+              for f in ('example4d.nii.gz', 'example4d.nii.gz')]
+    code, stdout, stderr = run_command(['nib-diff'] + fnames2, check_code=False)
+    assert_equal(stdout, "These files are identical.")
+
```

Comment:
what about comparing different files now?  at least a basic test using those nibabel/tests/data/spmT_0001*.nii.gz ?

Reply:
Don't you think that testing that get_headers_diff works is enough? My concern is that testing text output here has traditionally really messed up with Travis and AppVeyor because their Python environments for some reason print fewer significant figures than my computer does. What do you think?

Reply:
- in principle, you don't have to check output for 1-to-1 matching, you could check for the important segments to be there in the output (that fields you expect are listed, etc)
- ideally we should have output consistent across machines.
- as I've outlined in a separate email - if all the printing is moved to another function, then may be it should be tested in a dedicated to that function test. Here we just would need to check that script is indeed running and exiting as expected when we give the identical or different files, with only minimal testing of output (to verify that it is there)

Reply:
DONE